cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/scopes/Scope.kt

```diff
@@ -90,6 +90,9 @@ abstract class Scope(
      */
     @Transient var wildcardImports: MutableSet<ImportDeclaration> = mutableSetOf()
 
+    /** A list of scope modifications. */
+    @Transient var scopeModifications: MutableMap<Name, Scope> = mutableMapOf()
```

Comment:
this would be the alternative approach to the node. Depending on what we decide, this would be removed